### ISSUE TITLE
Add error when "to" prop is included on an SpsButton

### ISSUE THREAD

**Issue by natepreifwenner** [SPSCommerce/ui-react#45]
With the new SpsNavLinkButton component, we should throw an error if you include a "to" property to avoid devs expecting a Button with the "to" property to update the url/location.

**Comment by jimthedev** [comment]
Agreed. Do you think having it be a warning or error makes more sense?
